### PR TITLE
[PLAYER-4542] Controls look too big on mobile devices

### PR DESCRIPTION
**Pull request by marcosmx**
This branch contains the fix for restore the `responsiveClass` state at skin.js. Also mixes it with the `.oo-responsive` class.

### CODE REVIEW

**Comment by aeng7** [comment]
LGTM

**Comment by zelibobla** [comment]
LGTM

File: js/skin.js [hunk 1]
```diff
@@ -706,7 +706,7 @@ const Skin = createReactClass({
       }
     }
 
-    var className = ClassNames('oo-responsive', {
+    var className = ClassNames(this.state.responsiveClass, 'oo-responsive', {
```
Comment (KseniiaSokolovaEpam) [inline js/skin.js]: Where do you set value for responsiveClass state?

Comment (marcosmx) [reply js/skin.js]: @KseniiaSokolovaEpam that value comes from `ResponsiveManagerMixin` used as a mixin at the Skin component at line 32

**Pull request merged**